src/components/ManageAssets/ManageAssets.tsx

```diff
@@ -44,8 +63,8 @@ export const ManageAssets = ({ document }: ManageAssetsProps) => {
                 userWalletAddress={userWalletAddress}
                 approvedBeneficiaryAddress={approvedBeneficiaryAddress}
                 approvedHolderAddress={approvedHolderAddress}
-                beneficiaryAddress={beneficiaryAddress}
-                holderAddress={holderAddress}
+                beneficiaryAddress={beneficiary || ""}
+                holderAddress={holder || ""}
```

Comment:
um think related to the qn above, why approvedBeneficiaryAddress and approvedHolderAddress don't have to put the empty string fallback?

Reply:
added empty fallback.